logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -151,10 +151,14 @@ internal class ConnectionDataSource(
     ) {
         wrapStorageRequest {
             connections.forEach { connection ->
+                val qualifiedConversationId = idMapper.fromApiToDao(connection.qualifiedConversationId)
+                // If the conversation doesn't exists, we skip it, since the backend doesn't know this conv (probably status = CANCELLED)
+                conversationDAO.observeGetConversationByQualifiedID(qualifiedConversationId).first() ?: return
```

Comment:
Observing a flow just to read the first omitted value can be slow, and doing this for each connection request will add make the slow sync even slower (since BE will send all connections every single time)

Reply:
I think I got your point, I made a change to just let it fail for individual failed connections and not caring about if that's because the conversation didn't exist beforehand